pkg/operation/botanist/etcd.go

```diff
@@ -102,6 +105,14 @@ func (b *Botanist) DeployEtcd(ctx context.Context) error {
 		})
 	}
 
+	if b.Shoot.Purpose == gardencorev1beta1.ShootPurposeProduction {
+		var hvpaConfig = b.Shoot.Components.ControlPlane.EtcdMain.GetHVPAConfig()
+		if hvpaConfig != nil {
+			hvpaConfig.ScaleDownUpdateMode = pointer.StringPtr(hvpav1alpha1.UpdateModeOff)
+			b.Shoot.Components.ControlPlane.EtcdMain.SetHVPAConfig(hvpaConfig)
+		}
+	}
```

Comment:
Can you do this already in the `DefaultEtcd` function? Then you don't need to extend the interface with `GetHVPAConfig`?